### DISCUSSION TITLE
Do I need to re-run npm install at the root everytime I install a dependency in one of my apps?

### DISCUSSION THREAD

**Discussion by charlesloubao** [vercel/turborepo#253] [Help]
Hi.
I have a very simple setup for a monorepo:

```
apps/backend
packages/hello
```

`apps/backend` references my `hello` package like this
```
  "dependencies": {
    "hello": "*"
  },
```

I realized that if I install a package in my `apps/backend` folder for example `npm install express@latest`
`apps/backend/node_modules/hello` is overwritten with this package https://www.npmjs.com/package/hello

I have to go back to the root and run `npm install` for turborepo to map `hello` to my local package.

Do I have to do this everytime I install something into an app or am I doing something wrong?

You can find my test repo below
https://github.com/charleswritescode/monorepo-test

Thank you for any help 😃

**Comment by anthonyshew** [comment]
Closing as the reproduction no longer available!
